string  command
uint8   sensor
float32 param1
float32 param2
uint8   param3
---
bool    response
uint8   filter_size
float32 adc_sample_rate
uint8   range  